engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/dto/task/TaskDto.java

```diff
@@ -49,7 +50,9 @@ public class TaskDto {
   private String formKey;
   private CamundaFormRef camundaFormRef;
   private String tenantId;
-
+  @JsonProperty("attachment")
```

Comment:
Even I was reluctant to add, I am facing an weird issue where hasAttachment is translated to attachments instead of attachment in the json response, hence decided to add. Let me know your thoughts.

Reply:
I saw this. I will try to see from where it's coming.

Reply:
You can also try to debug it.

Reply:
I figured it , jackson considers the setter method to decide the boolean var name.